app/pathogen/arbovirus/dashboard/(visualizations)/median-seroprevalence-by-who-region-and-age-group-table.tsx

```diff
@@ -75,7 +76,7 @@ export const MedianSeroprevalenceByWhoRegionAndAgeGroupTable = () => {
                 ([whoRegion, dataPoints]) => [
                   whoRegion,
                   typedGroupBy(
-                    dataPoints,
+                    dataPoints.filter((dataPoint) => Object.values(AgeGroup).includes(dataPoint.ageGroup)),
```

Comment:
Some `nulls` were sneaking by, we'll just deal with them like this for now to make the table and the csv consistent :)